test/ctia/logger_test.clj

```diff
@@ -0,0 +1,36 @@
+(ns ctia.logger-test
+  (:require [ctia.logging :refer :all]
+            [ctia.events :as c :refer :all]
+            [clojure.test :as t :refer :all]
+            [schema.test :as st]
+            [clojure.tools.logging :as log]
+            [clojure.string :as str]))
+
+(use-fixtures :once st/validate-schemas)
+
+(deftest test-setup
+  (init!)
+  (let [{b :chan-buf c :chan m :mult :as ev} @central-channel]
+    (log-channel ev)
+    (send-create-event "tester" {} "TestModelType" {:data 1})
+    (Thread/sleep 100))) ;; wait until the go loop is done
+  
+(deftest test-logged
+  (init!)
+  (let [{b :chan-buf c :chan m :mult :as ev} @central-channel
+        sb (StringBuilder.)
+        ll log/log*
+        patched-log (fn [logger level throwable message]
+                      (.append sb message)
+                      (.append sb "\n"))]
+    (alter-var-root (var log/log*) (constantly patched-log))
```

Comment:
The alter-var-root is a smell to me.  If you can't use an atom, or can you use (with-redefs ...)?